test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/MiscCryptoSuite.java

```diff
@@ -67,6 +70,42 @@ public static void main(String... args) {
 		);
 	}
 
+	private HapiApiSpec sysAccountKeyUpdateBySpecialWontNeedNewKeyTxnSign(){
+		String sysAccount = "0.0.977";
+		String randomAccountA = "randomAccountA";
+		String randomAccountB = "randomAccountB";
+		String firstKey = "firstKey";
+		String secondKey = "secondKey";
+
+		return defaultHapiSpec("sysAccountKeyUpdateBySpecialWontNeedNewKeyTxnSign")
+				.given(
+						withOpContext((spec, opLog) -> {
```

Comment:
If we add the `.signedBy(GENESIS)` suggested below after L97, this `withOpContext(...)` will be unused, so we can remove it.